src/pages/HH_ManageHH.page

```diff
@@ -62,20 +62,30 @@
     
     <script>
     var j$ = jQuery.noConflict();
+    var ns = '{!namespacePrefix}';
+    var nsPrefix;
+    var nsPrefixEvent;
+    if (ns === '') {
+        nsPrefix = '';
+        nsPrefixEvent = 'c';
+    } else {
+        nsPrefix = ns + '__';
+        nsPrefixEvent = ns;
+    }
     
     var hhId = '{!hhId}';
     $Lightning.use("c:ManageHHApp", function() {
         $Lightning.createComponent("c:HH_Container", 
                                    { hhId : hhId,
-                                     namespacePrefix : "{!namespacePrefix}" },
+                                     namespacePrefix : nsPrefix },
                                    "lightning",
                                    function(cmp) {
                                        // after lightning is loaded, do what we need to do here!
                                        
                                        // hook up our event handlers
-                                       $A.eventService.addHandler({ "event": "c:HH_HouseholdSavedEvent", "handler" : saveAndCloseVisualforce});
-                                       $A.eventService.addHandler({ "event": "c:HH_ContactAfterDeleteEvent", "handler" : contactRemoved});
-                                       $A.eventService.addHandler({ "event": "c:HH_ContainerLoadedEvent", "handler" : doneLoading});
+                                       $A.eventService.addHandler({ "event": nsPrefixEvent + ":HH_HouseholdSavedEvent", "handler" : saveAndCloseVisualforce});
```

Comment:
does `c:eventName` not work here because it is being called from the `createComponent()` callback, or does `c:eventName` just not work for managed packages?  If it needs to be namespaced, I've got some updates to do elsewhere...